zap/src/output/luau/server.rs

```diff
@@ -888,6 +918,12 @@ impl<'a> ServerOutput<'a> {
 
 		self.push_studio();
 
+		self.push_line("local Players = game:GetService(\"Players\")");
```

Comment:
This is... awkward! Let's move this whole block of push functions you added into one function call maybe?